modules/ROOT/pages/x12-edi/x12-edi-connector.adoc

```diff
@@ -272,6 +272,9 @@ Other parser options let you:
 * Generate 999 Implementation Acknowledgments, rather than the 997 Functional Acknowledgments generated by default. Note that the support for 999 Implementation Acknowledgments does not include CTX segment generation.
 * Report segment error details to sender in the generated 997/999 acknowledgments.
 * Automatically include a standard 997 or 999 schema that matches the generated acknowledgment in the configuration for received messages.
+* Set the path to a Functional Acknowledgment schema to be used for generated acknowledgments such as `/hipaa/005010X231A1/999.esl` which uses the standard HIPAA 005010X231A1 acknowledgment.
```

Comment:
```suggestion
* Set the path to a Functional Acknowledgment schema to be used for generated acknowledgments such as `/hipaa/005010X231A1/999.esl`, which uses the standard HIPAA 005010X231A1 acknowledgment. +
```
This is not part of the green, changed text, but I wonder if there should be mention made in the first list about 997 being the default between 997 and 999?